test/performance/utility/detail/bits_of_benchmark.cpp

```diff
@@ -51,7 +51,7 @@ static void is_power_of_two_seqan3(benchmark::State & state) {
     for (auto _ : state)
     {
         n = std::rand();
-        benchmark::DoNotOptimize(seqan3::detail::is_power_of_two(n));
+        benchmark::DoNotOptimize(std::has_single_bit(n));
```

Comment:
please revert:

```suggestion
        benchmark::DoNotOptimize(seqan3::detail::is_power_of_two(n));
```

Reply:
Why? I thougt `is_power_of_two` and `bit_ceil` is now deprecated? And the whole file `bit_manipulation.hpp` aswell?